### DISCUSSION TITLE
Deployment and installation of updates as per state requirement

### DISCUSSION THREAD

**Discussion by manish050994** [Sunbird-cQube/community#845] [General]
Dear team,
As cQube is deployed in different state there are multiple new requirement and needs, according to this we are creating new modules.
But we are facing some issue while creating docker images, branching, installation and deployment. kindly provide a support and documentation. on the same.

**Comment by RNarmatha** [comment]
Hi manish050994,
In the [Microsite](https://cqube.sunbird.org/use-cqube/getting-started) , step by step guidance for the branching , installation and deployment. Please refer this  . 
We will connect with our DevOps team to get more details about creating the Docker images.

**Comment by RNarmatha** [comment]
Hi Manish,

Could you please help us to undertand your requirements on the support of creating the creating the docker Images. 
Kindly provide the requirements/ documentation on the support 
Thanks

**Comment by RNarmatha** [comment]
Hi Manish, 

Please refer the links for the following topics :

Branching and Installation - https://cqube.sunbird.org/use-cqube/getting-started
Deployment  process- https://cqube.sunbird.org/deployment-process/sdc-deployment ( Note: This link refers to SDC Deployment steps )
